src/pages/flows/[flow].tsx

```diff
@@ -1,20 +1,32 @@
 import Chat from "@pb/components/chat";
 import FlowBuilder from "@pb/components/flow-builder";
+import type { PBNodeProps } from "@pb/components/flow-builder/flow-builder.types";
```

Comment:
since it's a type file it will not have anything beside types
this is why I changed the type to be outside the braces instead of specifying the import object it self

Reply:
What I meant is that the flow that is applied within the project is to specify the type within the braces if we have only one type imported. Otherwise, it should be placed outside the braces when two or more types are imported.